viadot/sources/sap_rfc.py

```diff
@@ -438,7 +580,6 @@ def to_df(self):
             pd.DataFrame: A DataFrame representing the result of the query provided in `PyRFC.query()`.
         """
         params = self._query
-        columns = self.select_columns_aliased
```

Comment:
you cannot remove this part (and code below) because without it columns dont get aliased

Reply:
Yes, it can be, because there is no `try-except` sentence anymore, and then if nothing is in `report` the `df` is just populated with nothing but it isn't causing any error. Therefore, not needed lines from 493 to 496 (old code), and then the `columns` variable is useless.

Reply:
I mean this code doesn't give an error but it doesn't work as it's supposed to i.e. when you want to alias column in query "Select abcd as column1" the name of column 'abcd' in df doesnt change to 'column1'

Reply:
Now I understand what you want to point out with that. Changes done!